packages/payload/src/versions/getLatestGlobalVersion.ts

```diff
@@ -54,6 +53,9 @@ export const getLatestGlobalVersion = async ({
       globalExists,
     }
   }
+  if (!('createdAt' in latestVersion.version) || !('updatedAt' in latestVersion.version)) {
+    throw new Error('Could not find createdAt or updatedAt in latestVersion.version')
```

Comment:
Would this get properly caught? Does this need to throw `APIError` instead?

Reply:
What I'm doing doesn't actually change the runtime behavior other than a supposed error message.
If those properties don't currently exist, there would be an undefined error because an attempt is made to access them.
This is mostly to calm/silence TypeScript. I can change it to an APIError if you like.